layers/+lang/php/README.org

```diff
@@ -46,4 +46,4 @@ On any new project make sure to perform the following initialization tasks:
     touch .ac-php-conf.json
   #+END_SRC
 2. Inside of spacemacs run:
-  ac-php-remake-tags-all
+  = ac-php-remake-tags-all =
```

Comment:
If the spaces inside the `=` signs are removed, then the `=` signs won't be visible. The same way as
`=dotspacemacs-configuration-layers=` is written in the Install section.